src/shim/actors/convert.rs

```diff
@@ -0,0 +1,276 @@
+// Copyright 2019-2024 ChainSafe Systems
```

Comment:
Could you please explain why you copied https://github.com/ChainSafe/fil-actor-states/blob/main/fil_actor_interface/src/convert.rs here? If you intend to move it out of `fil-actor-states`, let's have a separate PR for this.

Reply:
These methods are not publicly exported.

Reply:
So right now we have two lengthy implementations doing basically the same thing. I'd rather we have only one.

Reply:
Fixed.

Reply:
Thank you!